guides/common/modules/proc_disabling-the-collection-of-usage-metrics.adoc

```diff
@@ -1,24 +1,39 @@
 [id="disabling-the-collection-of-usage-metrics"]
 = Disabling the collection of usage metrics
 
-You can opt out of the default usage metrics collection by masking the `systemd` timer that collects the data.
+You can opt out of the default usage metrics collection by masking the `systemd` timers that collect the data on your {ProjectServer}.
 
 .Procedure
-. On your {ProjectServer}, mask the `satellite-usage-metrics-generate.timer` unit:
+. To disable the collection of the full usage metrics:
+.. Mask the `satellite-usage-metrics-generate.timer` unit:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # systemctl mask satellite-usage-metrics-generate.timer
 ----
-. If the timer has been executed already at least once, remove the generated usage metrics file:
+.. If the timer has been executed already at least once, remove the generated usage metrics file:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # rm /var/lib/foreman-maintain/satellite_metrics.yml
 ----
+. To disable the collection of the condensed usage metrics:
+.. Mask the `satellite-usage-metrics-condensed.timer` unit:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# systemctl mask satellite-usage-metrics-generate.timer
```

Comment:
This command is identical to line 25 which looks wrong. See line 21.

Reply:
Oh right, good catch. The command now properly refers to the right timer.